perma_web/perma/templates/docs/accounts.html

```diff
@@ -0,0 +1,76 @@
+        <li class="reading-toc-section"><a href="#">Usage Limits</a></li>
+        <li class="reading-toc-chapter"><a href="#orgs">Organizations</a></li>
+        <li class="reading-toc-section"><a href="#free-service-academic">Free Service for Academic Organizations</a></li>
+        <li class="reading-toc-section"><a href="#free-service-courts">Free Service for Courts</a></li>
+        <li class="reading-toc-section"><a href="#paid-service">Paid Service for Other Organizations</a></li>
+        <li class="reading-toc-section"><a href="#authority">Authority and Responsibilities</a></li>
+        <li class="reading-toc-section"><a href="#adding-org-users">Adding and Removing Organization Users</a></li>
+      </ul>
+    </div>
+
+    <div class="col-sm-8 reading-body">
+      
+      <h2 class="body-ah">Accounts</h2>
+      <h3 id="who-needs-account" class="body-bh">Who Needs an Account?</h3>
+      <p class="body-text">To preserve web pages with Perma, you need an account. To register for a free, individual account, please visit our <a href="{% url 'sign_up' %}">signup page</a>.</p>
+      <p class="body-text">You don't need an account to view an archived page someone else made.</p>
+
+      <p></p>
+
+      <h3 id="becoming-org" class="body-bh">Usage Limits</h3>
+      <p class="body-text">Individual accounts let you preserve up to 10 web pages per month for free.</p>
+      <p class="body-text">To preserve more pages, you need to be affiliated with an organization that qualifies for free, unlimited service or one that has a paid subscription.</p>
+
+      <h2 class="body-ah">Organizational Accounts</h2>
+
+      <h3 id="free-service-academic" class="body-bh">Free Service for Academic Organizations</h3>
+      <p class="body-text">Free organizational accounts are available to students, faculty and academic journals if their academic library is a Perma.cc registrar. Please contact your library to find out if it is already a Perma.cc registrar. If not, your library can learn more and sign up for free using this page: <a href="{% url 'libraries' %}">https://perma.cc/libraries</a>.</p>
+      <p class="body-text">Paid accounts are currently available only for qualifying law firms. If you are interested, please contact us using this page: 
+
+      <h3 id="free-service-courts" class="body-bh">Free Service for Courts</h3>
+      <p class="body-text">Free organizational accounts also are available to state and federal courts. If you're affiliated with a court, you can use this page to learn more and contact us about signing up: <a href="{% url 'sign_up_courts' %}">https://perma.cc/signup/courts</a>.</p>
+
+      <h3 id="paid-service" class="body-bh">Paid Service for Other Organizations</h3>
+      <p class="body-text">Law firms, publishers, news organizations and others may obtain organizational accounts through a paid subscription. To learn more about paid subscriptions, please contact us at <a href="mailto: info@perma.cc">info@perma.cc</a>.</p>
```

Comment:
How about we point them to the contact form (`<a href="/contact?subject=Question%20About%20Paid%20Subscription">contact us</a>`), or to http://perma.dev/sign-up/firms here?